web_ui/src/pages/utils.ts

```diff
@@ -134,3 +136,11 @@ export const projectPointOnLine = ([startPoint, endPoint]: ProjectLine, point: P
         y: b.y * scale + startPoint.y,
     };
 };
+
+export const getMediaItemId = (item: MediaItem) => {
+    if (isImage(item)) {
+        return item.identifier.imageId;
+    }
+
+    return item.identifier.videoId;
```

Comment:
What if it is a video frame (I think it's possible that we receive video frames when working with the active set or some specific filters)? I believe we already have a utility that handles the three different type of media that we support.

Reply:
Right, it now uses the `getMediaId` utility, which handles both video and frame IDs. This code was removed in my last commit.